### PR TITLE
Clean ups

### PR DESCRIPTION
**Pull request by yanivel**
* Renames file names from camel case to snake case
* Changes some variable names & spacing
* Send the updateDots function to SwitchPageHelper instead of the widget

### CODE REVIEW

File: galaxy_mobile/lib/models/main_store.dart [hunk 1]
```diff
@@ -106,7 +106,7 @@ class MainStore extends ChangeNotifier {
     if (config == null) {
       config = await _api.fetchConfig();
     }
-    if( spec == null) {
+    if (spec == null) {
```
Comment (igala) [inline galaxy_mobile/lib/models/main_store.dart:109]: why did you remove this condition?

File: galaxy_mobile/lib/utils/switch_page_helper.dart [hunk 2]
```diff
@@ -26,27 +25,26 @@ class SwitchPageHelper {
       int pageSize,
       bool muteOtherCams,
       OnSwitchVideosCallback onSwitchVideosCallback) {
-    this.widget = widget;
-    pageSize = pageSize;
-    muteOtherCams = muteOtherCams;
-    makeSubscription = makeSubscription;
```
Comment (igala) [inline galaxy_mobile/lib/utils/switch_page_helper.dart]: why this field was removed? and also the one below?

File: galaxy_mobile/lib/models/main_store.dart (hunk 1 above)
Comment (yanivel) [reply galaxy_mobile/lib/models/main_store.dart:109]: Which condition? I didn't remove anything just modified spacing

File: galaxy_mobile/lib/utils/switch_page_helper.dart (hunk 2 above)
Comment (yanivel) [reply galaxy_mobile/lib/utils/switch_page_helper.dart]: I modified to use "this." prefix and changed the names of some of them.
I removed widget field and added onSwitchVideosCallback instead.
The 6 fields are still there
